ambry-store/src/test/java/com/github/ambry/store/StorageManagerTest.java

```diff
@@ -269,6 +270,131 @@ public void scheduleAndControlCompactionTest() throws Exception {
     shutdownAndAssertStoresInaccessible(storageManager, replicas);
   }
 
+  /**
+   * Helper util to add blobs to a given Store
+   * @param store store to add blob to.
+   * @param size size in bytes of the randomized blob
+   * @param expiresAtMs expiry in milliseconds to be set for the blob
+   * @return {@link MockId} the mock id of the blob added
+   * @throws StoreException
+   */
+  public MockId addRandomBlobToStore(Store store, long size, long expiresAtMs) throws StoreException {
```

Comment:
Can we add test for blob removal as well, just to make sure that the new addition is compatible with the removal flow.

Reply:
More tests for all negative conditions for all exceptions thrown at Disk Manager, Storage Manager and Blob Store Addition + removal.

Reply:
@aga9900 I will be adding a TODO for additional tests since we will have new tests once FC and prefilecopy flow done as well.